eurobot_loc/launch/pf_launch.launch

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<launch>
+  <rosparam file="$(find eurobot_loc)/config/secondary_robot_params.yaml" command="load" ns="secondary_robot"/>
+  <param name = "field/color" value = "orange"/>
```

Comment:
Change this, create a topic for sending start side, make default start side in config